### DISCUSSION TITLE
"Meditate" e.g. using religious items to remove stacks of Tainted Mind

### DISCUSSION THREAD

**Discussion by Salty-Panda** [CleverRaven/Cataclysm-DDA#45836] [Ideas]
# Is your feature request related to a problem? Please describe.

Got banged upon by very bad dogs and had no idea why.
Discovered that the cause was a little eye winking at me. Managed to escape and had a little idea to try to use a small relic I wear (I heard it might give some mana boost) to meditate. It did a whole lot of nothing.

# Describe the solution you'd like

A way to clear the mind, either natural (religion), technological (tinfoil hat or CBM), mutation (some of the branches ARE fishy 
and could mesh well with such ability), or magical (Magus class would be fitting for such spell).
Could be separated into some preemptive measure and a method to reduce the problem

# Describe alternatives you've considered

Checking if a combination of a blindfold and IR CBM would work.
Checking if throwing salt at doggos does something.

**Comment by kevingranade** [comment]
In vanilla, this is AFAICT only applied by flaming eyes. Religious thinking would have no effect on this.
A mod can certainly do so, but it needs to be localized to that mod.
